lib/src/apivideo_player.dart

```diff
@@ -1,4 +1,4 @@
-import 'package:apivideo_player/src/apivideo_player_overlay.dart';
+import 'package:apivideo_player/src/views/apivideo_player_overlay.dart';
```

Comment:
instead of `views`. A `widgets` folder would be better.
And this player widget have definitely his places in the `widgets` folder.